# generated by patch-package 6.4.10
#
# command:
#   npx patch-package node-gitlab
#
# declared package:
#   node-gitlab: ^1.6.1
#
diff --git a/node_modules/node-gitlab/lib/resources/index.js b/node_modules/node-gitlab/lib/resources/index.js
index 7f67a0a..a37e559 100644
--- a/node_modules/node-gitlab/lib/resources/index.js
+++ b/node_modules/node-gitlab/lib/resources/index.js
@@ -26,6 +26,8 @@ module.exports = {
 
   repositoryBranches: require('./repository_branch'),
 
+  repositoryTags: require('./repository_tag'),
+
   repositoryFiles: {
     resourcePath: '/projects/:id/repository/files'
   },
diff --git a/node_modules/node-gitlab/lib/resources/repository_tag.js b/node_modules/node-gitlab/lib/resources/repository_tag.js
new file mode 100644
index 0000000..f3f5498
--- /dev/null
+++ b/node_modules/node-gitlab/lib/resources/repository_tag.js
@@ -0,0 +1,22 @@
+'use strict';
+
+var util = require('util')
+var restful = require('restful-client')
+
+function Tag(client) {
+  this.constructor.super_.call(this, client, '/projects/:id/repository/tags', 'tag_name');
+}
+
+util.inherits(Tag, restful.RESTFulResource);
+
+Tag.prototype.createRelease = function (params, callback) {
+  this.client.request('post', this.onePath + '/release', params, callback);
+  return this;
+};
+
+Tag.prototype.updateRelease = function (params, callback) {
+  this.client.request('put', this.onePath + '/release', params, callback);
+  return this;
+};
+
+module.exports = Tag;
